core/create/src/index.ts

```diff
@@ -139,7 +139,9 @@ async function executeMigration(deleteConfig: boolean) {
 
   packageJson.writeChanges()
 
-  const installPromise = logger.logPromise(exec('npm install'), 'installing dependencies')
+  await logger.logPromise(exec('npm install'), 'installing dependencies')
+  // we need to import installHooks from the app itself instead of npx or else loadPlugin will load rawPlugin from npx and Task will be loaded from the app, leading to task.prototype failing the instanceof Task check
+  const installHooks: any = ((await importFrom(process.cwd(), 'dotcom-tool-kit/lib/install')) as any).default
```

Comment:
**question**: is is possible to cast this to the actual type of `installHooks`? even if we're doing `as unknown as typeof installHooks` i think that would be better?

**nitpick**: `importFrom` is synchronous so the `await` is unnecessary (same below)